presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/BeginTableWrite.java

```diff
@@ -210,8 +210,9 @@ private PlanNode rewriteDeleteTableScan(PlanNode node, TableHandle handle)
                         layoutResult.getLayout().getNewTableHandle(),
                         scan.getOutputVariables(),
                         scan.getAssignments(),
-                        layoutResult.getLayout().getPredicate(),
-                        computeEnforced(originalEnforcedConstraint, layoutResult.getUnenforcedConstraint()));
+                        scan.isDelete(),
```

Comment:
Where the `isDelete` getter defined?

Reply:
gah, thanks. that was the old name, probably didn't squash the change into the right commit.